core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -301,4 +305,15 @@ protected static String fullTableName(String catalogName, TableIdentifier identi
 
     return sb.toString();
   }
+
+  private synchronized MetricsReporter metricsReporter() {
+    if (metricsReporter == null) {
+      metricsReporter =
+          properties().containsKey(CatalogProperties.METRICS_REPORTER_IMPL)
+              ? CatalogUtil.loadMetricsReporter(
+                  properties().get(CatalogProperties.METRICS_REPORTER_IMPL))
+              : LoggingMetricsReporter.instance();
+    }
+    return metricsReporter;
```

Comment:
nit: newline missing after the closing if stmt

Reply:
added newline.